build/docker-compose.common.yml

```diff
@@ -1,7 +1,7 @@
 version: '2.4'
 services:
   mysql:
-    image: "mysql:5.7"
+    image: "mysql:5.6"
```

Comment:
why the version change?

Reply:
We want to ensure that we are using in our dev environment (And in the CI) the oldest supported version. We have had problems with features that didn't work with MySQL 5.6 because we were using 5.7 in development and some features were not available.